bin/prep-release

```diff
@@ -34,12 +34,15 @@ git commit -m "Release v$version"
 git push origin $branch
 
 compare_link="https://github.com/codeclimate/codeclimate/compare/${old_version}...$(git rev-parse --short $branch)"
+pr_description_file=`mktemp -t cc_commit_message` || exit 1
```

Comment:
I borrowed this from an example online, but my understanding was that `$(…)` could hide a bad exit code? I can never remember how `$(…)` and <code>\`…\`</code> differ.